dataverse-webapp/src/main/webapp/dashboard-licenses.xhtml

```diff
@@ -11,6 +11,7 @@
 
 <h:body>
     <f:metadata>
+        <ui:param name="pageTitle" value="#{bundle['dashboard.license.header']}" />
         <f:viewParam name="dataverseId" value="#{dashboardPage.dataverseId}"/>
         <f:viewAction action="#{dataverseSession.updateLocaleInViewRoot}"/>
         <f:viewAction action="#{LicenseListingPage.init}"/>
```

Comment:
To be consistent with the rest of the application we should probably change the look of add license and reorder licenses buttons. From what I can see we are missing icons and the padding settings are different than in the rest of the app. Button for add should look like every button where we are adding something (plus icon and text). For reorder we should have the same icon as in reorder files on dataset page

Reply:
Done. This necessiated changing `<p:commandButton>` to `<p:commandLink>`, because as far as I could gather, the `commandButton` does not allow adding HTML tags as its value. While the `commandButton` supports an `icon` attribute, it was designed to be used with FontAwesome icon pack, while our app relies on Glyphicon.